_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -38,11 +41,16 @@ The `object` element is not rendered for presentational purposes. If the `object
 
 ## Accessibility Support
 
-Non-supported media formats make screen readers render the text content of the element instead of other attributes.
+Non-supported media formats make screen readers render the text content of the element instead of other attributes. `object` elements without an accessible name are ignored by assistive technologies unless they have an [explicit role][].
 
 ## Background
 
-Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately.
+This rule is testing that the following 'object' attributes are followed:
```

Comment:
I don't think "attributes" is the correct word. The list does not really mention attributes (in HTML sense of the term) of the `object` element. The mix-up with the technical language is a bit weird.